package.json

```diff
@@ -24,6 +24,7 @@
     "qs": "^6.10.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-dropzone": "^14.2.3",
```

Comment:
I thought dropzones ended up being unneeded?

Reply:
you are correct! I'll remove